<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>390</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_up">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>360</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>200</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_down">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>128</width>
        <height>96</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>depth image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_save_picture">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>139</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_color_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_color_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_color_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_depth_1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>96</y>
        <width>128</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_depth_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>116</y>
        <width>128</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="button_stop_save">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <property name="text">
       <string>stop save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_add_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <property name="text">
       <string>add label</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
